### PR TITLE
Change warning on the profile page

### LINKED ISSUE #2513: Change warning on the profile page when connected to Rinkeby/GC
1. Connect to a wallet in Rinkeby/GC
2. Open Profile page
3. Check the warning message

**AR**: 'Profile data is only available for mainnet. Please change the network to see it.' message is displayed

**ER**: Change to  '**Affiliate data is only available for Ethereum. Please change the network to see it.**' as everywhere in the app

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Fixes #2513 
image
  # To Test

1. Connect to a wallet in Rinkeby/GC
2. Open Profile page
3. Check the warning message. Should say: '**Affiliate** data is only available for **Ethereum**. Please change the network to see it.'

### CODE REVIEW

File: src/hooks/useFetchListCallback.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export function useFetchListCallback(): (listUrl: string, sendDispatch?: boolean
         if (networkLibrary && network.chainId === 1) {
           return resolveENSContentHash(ensName, networkLibrary)
         }
-        throw new Error('Could not construct mainnet ENS resolver')
+        throw new Error('Could not construct Ethereum ENS resolver')
```
Comment (ramirotw) [inline src/hooks/useFetchListCallback.ts:24]: this isn't inside the `custom` folder and shouldn't be modified

**Review by ramirotw — CHANGES_REQUESTED** [review]

**Comment by anxolin** [comment]
I'm confused with the changes. Mainnet is the right name isn't it, or if we want to add Ethereum, would be Ethereum Mainnet. @elena-zh what is the reason to call it just "Ethereum"?

**Comment by elena-zh** [comment]
@anxolin , this is due to we use 'Ethereum' in the network switcher. 

The same is on Explorer. 
So I thought we should be consistent with networks naming everywhere.

**Review by elena-zh — APPROVED** [review]
LGTM!

**Review by ramirotw — APPROVED** [review]

**Comment by anxolin** [comment]
Good points Elena, yes we should be consistent

**Pull request merged**